app/experimenter/experiments/email.py

```diff
@@ -66,3 +67,38 @@ def send_experiment_launch_email(experiment):
     )
 
     email.send(fail_silently=False)
+
+
+def send_experiment_ending_email(experiment):
+
+    html_content = render_to_string(
+        "experiments/emails/experiment_ending_email.html",
+        {
+            "experiment": experiment,
+            "change_window_url": ExperimentConstants.NORMANDY_CHANGE_WINDOW,
+        },
+    )
+
+    version = experiment.format_firefox_versions
```

Comment:
What purpose do these two lines serve? Why not just inline these where they're used on line 91?

Reply:
i'll inline them, there was no advantage.